### PR TITLE
Update azure_rm_cosmosdbaccount related module document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_cosmosdbaccount related module document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_cosmosdbaccount/azure_rm_cosmosdbaccount_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     location:
         description:
             - The location of the resource group to which the resource belongs.
-            - Required when C(state) is I(present).
+            - Required when I(state) is C(present).
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount.py]: Required when I(state=present).

File: lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount.py [hunk 2]
```diff
@@ -57,44 +57,46 @@
                     - 'consistent_prefix'
             max_staleness_prefix:
                 description:
-                    - "When used with the Bounded Staleness consistency level, this value represents the number of stale requests tolerated. Accepted range
-                       for this value is 1 - 2,147,483,647. Required when I(default_consistency_policy) is set to C(bounded_staleness)."
+                    - When used with the Bounded Staleness consistency level, this value represents the number of stale requests tolerated.
+                    - Accepted range for this value is 1 - 2,147,483,647. Required when I(default_consistency_policy) is set to C(bounded_staleness).
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount.py]: Required when I(default_consistency_policy=bounded_staleness).

**Review by yuwzho — CHANGES_REQUESTED** [review]

**Review by yuwzho — APPROVED** [review]
shipit

**Comment by yungezz** [comment]
shipit

**Comment by acozine** [comment]
Looks like some options in the argspec are missing or incorrectly documented.

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount_facts.py [hunk 3]
```diff
@@ -125,7 +125,7 @@
                     sample: 100
         failover_policies:
             description:
-                - Read locations
+                - Read locations.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount_facts.py]: This doesn't seem right. Shouldn't this be "Failover policies" or something like that?

File: lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount_facts.py [hunk 4]
```diff
@@ -339,10 +339,10 @@
                              q3dpJxJga76h9BZkK2BJJrDzSO6DDn6yKads017OZBZ1YZWyq1cW4iuvA=="
         tags:
             description:
-                - Tags
+                - Tags assigned to the resource. Dictionary of string:string pairs.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount_facts.py]: I would put the quotes in here as well, like this:

```suggestion
                - Tags assigned to the resource. Dictionary of "string":"string" pairs.
```

File: lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount_facts.py [hunk 5]
```diff
@@ -339,10 +339,10 @@
...
             returned: always
-            type: complex
-            sample: {}
+            type: dict
+            sample: { "tag1": "abc" }
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount_facts.py]: I can't tell if there should be a space or not. If there should be a space, then add one in line 342 above (`"string": "string"`). If there should NOT be a space, then remove it here (`"tag1":"abc"`).

**Review by acozine — COMMENTED** [review]
Couple of small comments.

**Pull request merged**